libs/sql-parser/src/main/antlr/io/crate/sql/parser/antlr/SqlBaseParser.g4

```diff
@@ -913,6 +914,7 @@ nonReserved
     | COPY
     | CURRENT
     | CURRENT_DATE
+    | CURRENT_CATALOG
```

Comment:
Isn't it reserved now?

Reply:
yep, see the discussion here: https://github.com/crate/crate/pull/17972#pullrequestreview-2896603953, it's also reserved for Postgres: https://www.postgresql.org/docs/current/sql-keywords-appendix.html

Reply:
But it's added here to the non-reserved ones.

Reply:
yes, it' leftover, sorry, I mixed things up, removed from the list in the grammar.

Reply:
Apologies, I rushed here and merged the PR, it is required to be in the `nonReserved` if it's registered also as a normal function call `()`, otherwise the parser doesn't allow such calls, but since PostgreSQL doesn't do that (the `()` registration, but only as keyword): https://github.com/crate/crate/pull/18007
so we avoid having a confusing entry in the `nonReserved` list in the grammar.